pronto_quadruped_ros/src/legodo_handler_ros.cpp

```diff
@@ -120,8 +128,8 @@ void LegodoHandlerBase::getPreviousState(const StateEstimator *est)
         dl_pose_->addSampleCSV(time, head_state_.position(), orientation_);
         dl_vel_->addSample(time, head_state_.velocity(), head_state_.angularVelocity());
 
-        Matrix3d vel_cov = head_cov_.block<3,3>(RBIS::velocity_ind, RBIS::velocity_ind);
-        Matrix3d omega_cov = head_cov_.block<3,3>(RBIS::angular_velocity_ind, RBIS::angular_velocity_ind);
+        Eigen::Block<RBIM, 3, 3> vel_cov = head_cov_.block<3,3>(RBIS::velocity_ind, RBIS::velocity_ind);
```

Comment:
what's the advantage of the `Eigen:Block` thing?

Reply:
It's a reference to the block rather than creating a copy on the heap